schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals
+(
+	id int GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] [OPTIONAL] It'd we a good idea if you make the `id` column the primary key.

Reply:
Alright, Thank you for the comment.